Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushNotificationBus.h

```diff
@@ -38,23 +38,33 @@ namespace AzToolsFramework
             AZStd::span<float> opacities,
             AZStd::span<bool> validFlags)>;
 
-        //! OnIntensityChanged notifies listeners that the paintbrush intensity setting has changed.
-        //! @param intensity The new intensity setting for the paintbrush (0=black, 1=white).
-        virtual void OnIntensityChanged([[maybe_unused]] float intensity) { }
-
-        //! OnOpacityChanged notifies listeners that the paintbrush opacity setting has changed.
-        //! @param opacity The new opacity setting for the paintbrush (0=transparent, 1=opaque).
-        virtual void OnOpacityChanged([[maybe_unused]] float opacity) { }
-
-        //! OnRadiusChanged notifies listeners that the paintbrush radius setting has changed.
-        //! @param radius The new radius setting for the paintbrush, in meters.
-        virtual void OnRadiusChanged([[maybe_unused]] float radius) { }
-
         //! OnWorldSpaceChanged notifies listeners that the paintbrush transform has changed,
         //! typically due to the brush moving around in world space.
         //! This will get called in each frame that the brush transform changes.
         //! @param brushTransform The new transform for the brush position/rotation/scale.
-        virtual void OnWorldSpaceChanged([[maybe_unused]] const AZ::Transform& brushTransform) { }
+        virtual void OnWorldSpaceChanged([[maybe_unused]] const AZ::Transform& brushTransform)
+        {
+        }
+
+        //! OnPaintModeBegin notifies listeners that the paint mode has been entered.
+        virtual void OnPaintModeBegin()
+        {
+        }
+
+        //! OnPaintModeEnd notifies listeners that the paint mode is exiting.
+        virtual void OnPaintModeEnd()
+        {
+        }
+
+        //! OnPaintBegin notifies listeners that painting has begun.
```

Comment:
nit: For these API comments you could remove the repeated function name and just start with `Notifies..`